### PR TITLE
feat: update react query version

### PR DESCRIPTION
**Pull request by ShinySyntax**

### CODE REVIEW

File: apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts [hunk 1]
```diff
@@ -39,20 +39,16 @@ export class HonoErrorHandlerService {
   }
 
   private async reportError<E extends Env = any>(error: Error, c: Context<E>): Promise<void> {
-    try {
-      const id = this.sentry.captureEvent(await this.getSentryEvent(error, c));
-      this.logger.info({ event: "SENTRY_EVENT_REPORTED", id });
-    } catch (e) {
-      this.logger.error(e);
-    }
+    const id = this.sentry.captureEvent(await this.getSentryEvent(error, c));
```
Comment (baktun14) [inline apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts]: Any reason why you removed the try catch?

File: apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts [hunk 2]
```diff
@@ -39,20 +39,16 @@ export class HonoErrorHandlerService {
...
+    this.logger.info({ event: "SENTRY_EVENT_REPORTED", id });
   }
 
   private async getSentryEvent<E extends Env = any>(error: Error, c: Context<E>): Promise<Event> {
     const event = this.sentry.addRequestDataToEvent(this.sentryEventService.toEvent(error), {
       method: c.req.method,
       url: c.req.url,
       headers: omit(Object.fromEntries(c.req.raw.headers), ["x-anonymous-user-id"]),
-      body: await this.getSentryEventRequestBody(c)
+      body: await c.req.json()
```
Comment (baktun14) [inline apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts]: Same question here?

File: apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts [hunk 3]
```diff
@@ -69,18 +65,4 @@ export class HonoErrorHandlerService {
 
     return event;
   }
-
-  private async getSentryEventRequestBody<E extends Env = any>(c: Context<E>) {
```
Comment (baktun14) [inline apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts]: ?

**Review by baktun14 — CHANGES_REQUESTED** [review]
I think you need to rebase your branch/fork because some changes are being unmade by your pr.

File: apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts (hunk 1 above)
Comment (ShinySyntax) [reply apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts]: I added.

File: apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts (hunk 3 above)
Comment (ShinySyntax) [reply apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts]: I added.

**Comment by ShinySyntax** [comment]
I fixed all

File: apps/api/package.json [hunk 4]
```diff
@@ -36,7 +36,7 @@
     "@akashnetwork/database": "*",
     "@akashnetwork/http-sdk": "*",
     "@casl/ability": "^6.7.1",
-    "@chain-registry/assets": "^1.64.79",
+    "@chain-registry/assets": "^0.7.1",
```
Comment (baktun14) [inline apps/api/package.json:39]: Why change the version?

File: apps/api/src/utils/coin.ts [hunk 5]
```diff
@@ -1,55 +1,48 @@
 import { asset_lists } from "@chain-registry/assets";
```
Comment (baktun14) [inline apps/api/src/utils/coin.ts:1]: Why change this file?

File: apps/api/src/routes/v1/deployments/byOwnerDseq.ts [hunk 6]
```diff
@@ -15,9 +15,8 @@ const route = createRoute({
         description: "Owner's Address",
```
Comment (baktun14) [inline apps/api/src/routes/v1/deployments/byOwnerDseq.ts:15]: why change this file?

File: apps/api/src/routes/internal/leasesDuration.ts [hunk 7]
```diff
@@ -15,7 +15,7 @@ const route = createRoute({
       owner: z.string().openapi({ example: openApiExampleAddress })
```
Comment (baktun14) [inline apps/api/src/routes/internal/leasesDuration.ts:15]: why change this file?

File: apps/deploy-web/package.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "akash-console",
-  "version": "2.13.0",
+  "version": "2.12.0",
```
Comment (baktun14) [inline apps/deploy-web/package.json:3]: ?

File: apps/deploy-web/src/components/new-deployment/ManifestEdit.tsx [hunk 9]
```diff
@@ -83,7 +83,7 @@ export const ManifestEdit: React.FunctionComponent<Props> = ({ editedManifest, s
   useWhen(
     wallet.isManaged && sdlDenom === "uakt",
     () => {
-      setEditedManifest(prev => (prev ? prev.replace(/uakt/g, managedDenom) : prev));
+      setEditedManifest(prev => prev.replace(/uakt/g, managedDenom));
```
Comment (baktun14) [inline apps/deploy-web/src/components/new-deployment/ManifestEdit.tsx:86]: Why change this file?

File: apps/provider-console/src/context/SettingsProvider/SettingsProviderContext.tsx [hunk 10]
```diff
@@ -65,83 +65,97 @@ export const SettingsProvider: FC<{ children: ReactNode }> = ({ children }) => {
...
-      const currentNetwork = networks.find(x => x.id === _selectedNetworkId);
-      const response = await axios.get(currentNetwork?.nodesUrl || mainnetNodes);
-      const nodes = response.data as Array<{ id: string; api: string; rpc: string }>;
-      const mappedNodes: Array<BlockchainNode> = await Promise.all(
-        nodes.map(async node => {
-          const nodeStatus = await loadNodeStatus(node.rpc);
-
-          return {
-            ...node,
+      try {
```
Comment (baktun14) [inline apps/provider-console/src/context/SettingsProvider/SettingsProviderContext.tsx:68]: Why change this file?

**Review by baktun14 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**